src/main/groovy/org/shipkit/internal/gradle/GitPlugin.java

```diff
@@ -111,9 +111,9 @@ public void execute(final Task t) {
 
         TaskMaker.execTask(project, COMMIT_CLEANUP_TASK, new Action<Exec>() {
             public void execute(final Exec t) {
-                t.setDescription("Removes last commit, using 'reset --hard HEAD~'");
+                t.setDescription("Removes last commit, using 'reset --soft HEAD~'");
```

Comment:
I feel you ;) Happened to me multiple times! We should go ahead an implement the TODO, ```git reset --hard``` is pretty brutal.